packages/xo-server-load-balancer/src/density-plan.js

```diff
@@ -208,7 +208,11 @@ export default class DensityPlan extends Plan {
         debug(
           `Migrate VM (${vm.id} "${vm.name_label}") to Host (${destination.id} "${destination.name_label}") from Host (${fmtSrcHost}).`
         )
-        return xapiDest.migrateVm(vm._xapiId, this.xo.getXapi(destination), destination._xapiId)
+        return this._concurrentMigrationLimiter(xapiDest.migrateVm.bind(xapiDest))(
```

Comment:
Why is that necessary? `xo-server/xapi#migrateVm` already has a limit on migrations?

Reply:
The concurrency limiting on migrations that already exists on `xo-server/xapi` only applies to migrations with storage motion. According to Olivier, for the moment it is better to not apply this limitation to migrations that do not use storage motion and are made outside of the load balancer.

Reply:
After discussion with @olivierlambert, I agree we need to this limitation in the plugin.

If there are issues in the future due to interaction between this and the built-in XO limitation, we will uncouple them.